packages/transformers/babel/src/env.js

```diff
@@ -1,4 +1,4 @@
-// @flow
+// @flow strict-local
 
 import type {Config} from '@parcel/types';
 import type {BabelTargets} from './types';
```

Comment:
We can remove the existing definition of `BabelTargets` from this types file and use the new one 😄